exercises/square-root/description.md

```diff
@@ -0,0 +1,7 @@
+Given a natural radicand, return its square root.
```

Comment:
Maybe the word "radicand" needs clarification in one way or another.

I didn't know this term and felt kinda stupid. :-)

Maybe we should simply write "natural number"?

After all, the text below implies that those terms can be interchanged.

```suggestion
Given a natural number, return its square root.
```

Reply:
Can add a description of the term to the `readme.md` instead. Removing radicand means removing it from the `canonical-data.json` also, where the alternatives are clumsy, not very descriptive or not very distinguishable from any other numerical method argument  (e.g. `number`, `naturalNumber`, `input`, etc)

Reply:
Just as fine by me.